model/common.go

```diff
@@ -1,13 +1,13 @@
 package model
 
 type Swagger struct {
-	Swagger string `json:"swagger"`
-	Info    Info   `json:"info"`
-	Host    string `json:"host"`
-	BasePath string `json:"basePath,omitempty"`
-	Schemes []string `json:"schemes,omitempty"`
-	Paths   map[string]*PathItem `json:"paths"` // Ignoring extensions for now
-	Definitions map[string]*Schema `json:"definitions"`
+	Swagger     string               `json:"swagger"`
+	Info        Info                 `json:"info"`
+	Host        string               `json:"host"`
+	BasePath    string               `json:"basePath,omitempty"`
+	Schemes     []string             `json:"schemes,omitempty"`
+	Paths       map[string]*PathItem `json:"paths"` // Ignoring extensions for now
```

Comment:
I have been wondering about that. Since we are comparing valid jsons, my solution was to unmarshal the provided jsons and compare them with the generated jsons with DeepEqual.